spec/features/merchants/items/index_spec.rb

```diff
@@ -39,13 +20,15 @@
     visit "/merchants/#{@merchant_1.id}/items"
 
     expect(page).to have_content(@item_1.name)
-    expect(page).to_not have_content(@item_4.name)
+    expect(page).to_not have_content(@item_3.name)
   end
 
   it 'links to item show page' do
     visit "/merchants/#{@merchant_1.id}/items"
 
-    click_on "#{@item_1.name}"
+    within "#id-#{@item_1.id}" do
```

Comment:
I've only seen this syntax used before for the within block, but if it works it works